test/e2e/e2e_ginkgo/e2e_ginkgo_suite_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e_ginkgo
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+
+	v1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
```

Comment:
nit: group your import as:
import (
built-in

external

local
)
or at least group them.

Reply:
Done